drape_frontend/frontend_renderer.cpp

```diff
@@ -1320,10 +1320,11 @@ void FrontendRenderer::ProcessSelection(ref_ptr<SelectObjectMessage> msg)
   if (msg->IsDismiss())
   {
     m_selectionShape->Hide();
-    if (!m_myPositionController->IsModeChangeViewport() && m_selectionTrackInfo.is_initialized())
+    if (!m_myPositionController->IsModeChangeViewport() && m_selectionTrackInfo.has_value())
```

Comment:
without has_value?

Reply:
Why is it better?
I preferred to leave has_value in the places where the original authors used is_initialized (boost::optional has operator bool too).

Reply:
okey